_staffers/FahadFarid.md

```diff
@@ -0,0 +1,8 @@
+---
+name: Fahad Farid
+role: Teaching Assistant
+email: fahadfn@bu.edu
+photo: fahad_farid.jpg
+---
+
+Office Hours: Wednesday 5.15-6.15PM Location: TBD
```

Comment:
@fahdfareed  this conflicts with Lecture (4:30 - 5:45). Are you available Tuesday or earlier in the day wednesday?

Reply:
changes made